pkg/madmin/API.md

```diff
@@ -36,15 +36,12 @@ func main() {
 
 ```
 
-| Service operations|LockInfo operations|Healing operations|Config operations| Misc |
-|:---|:---|:---|:---|:---|
-|[`ServiceStatus`](#ServiceStatus)| [`ListLocks`](#ListLocks)| [`ListObjectsHeal`](#ListObjectsHeal)|[`GetConfig`](#GetConfig)| [`SetCredentials`](#SetCredentials)|
-|[`ServiceRestart`](#ServiceRestart)| [`ClearLocks`](#ClearLocks)| [`ListBucketsHeal`](#ListBucketsHeal)|[`SetConfig`](#SetConfig)||
-| | |[`HealBucket`](#HealBucket) |||
-| | |[`HealObject`](#HealObject)|||
-| | |[`HealFormat`](#HealFormat)|||
-| | |[`ListUploadsHeal`](#ListUploadsHeal)|||
-| | |[`HealUpload`](#HealUpload)|||
+| Service operations                  | LockInfo operations         | Healing operations                    | Config operations         | Misc                                |
+|:------------------------------------|:----------------------------|:--------------------------------------|:--------------------------|:------------------------------------|
+| [`ServiceStatus`](#ServiceStatus)   | [`ListLocks`](#ListLocks)   | [`HealStart`](#HealStart)             | [`GetConfig`](#GetConfig) | [`SetCredentials`](#SetCredentials) |
+| [`ServiceSendAction`](#ServiceSendAction) | [`ClearLocks`](#ClearLocks) | [`HealStatus`](#HealStatus)           | [`SetConfig`](#SetConfig) |                                     |
+|                                     |                             | [`HealStop`](#HealStop)               |                           |                                     |
```

Comment:
`HealStop` isn't implemented, should we remove it?

Reply:
Yes, will be removing this.